### DISCUSSION TITLE
About the optimization of G-Helper window

### DISCUSSION THREAD

**Discussion by Tante2000** [seerge/g-helper#701] [Ideas]
It is recommended to make the G-Helper window similar to the status window of Windows. Click the application icon to pop up the window, and click the position outside the application window to automatically close the window, so that I don’t have to manually click the close button after adjusting some functions of G-Helper. Efficiency can be greatly improved

**Comment by seerge** [comment]
@Tante2000 i don't really like this idea, as people seem to monitor some things like temps / fan speeds while keeping window open.

Did you know what app also has a tray-icon and right-click context menu (with all main modes) that hides immediately after you select something there? :)

**Comment by Tante2000** [comment]
https://twinkletray.com/
This app is very good
Of course, maybe not everyone likes this feature, but we can make this feature optional, so it's perfect
